nncf/quantization/algorithms/post_training/algorithm.py

```diff
@@ -189,13 +191,15 @@ def _create_statistics_aggregator(self, dataset: Dataset, backend: BackendType)
             return PTStatisticsAggregator(dataset)
         return None
 
-    def _apply(
+    def apply(
```

Comment:
Could you please update `PostTrainingQuantization.get_statistic_points()` method as well?

Reply:
Thanks for the comment. I updated `PostTrainingQuantization.get_statistic_points()` taking into account PR https://github.com/openvinotoolkit/nncf/pull/2013, because `PostTrainingQuantization.get_statistic_points()` function has incorrect implementation anyway.

cc @daniil-lyakhov